docs/rhel-custom-image.md

```diff
@@ -0,0 +1,57 @@
+# RedHat Enterprise Linux
+
+Cloud providers which are listed below, support using rhel as an operating system option: 
+
+- AWS 
+- Azure
+- GCE
+- KubeVirt
+- Openstack
+- vSphere
+
+####  AWS:
+For amazon web service cloud provider, one the user applies for Gold Accesses Images using their RedHat Customer Portal, their request 
```

Comment:
I would streamline a bit the sentence and add a link to https://access.redhat.com/public-cloud/aws, something like:

```
First of all the RHEL gold image AMIs have to be enabled from the [RedHat Customer Portal](https://access.redhat.com/public-cloud/aws) (this requires a [cloud-provider subscription](https://access.redhat.com/public-cloud)).
```